src/graph/components/runtime.tsx

```diff
@@ -0,0 +1,85 @@
+import React, { useEffect, useState } from "react";
+import { IAuthoredState } from "./types";
+import {
+  addLinkedInteractiveStateListener, IInteractiveStateWithDataset, removeLinkedInteractiveStateListener, IDataset
+} from "@concord-consortium/lara-interactive-api";
+import { Bar } from "react-chartjs-2";
+import { generateChartData, emptyChartData } from "../generate-chart-data";
+
+export interface IProps {
+  authoredState: IAuthoredState;
+}
+
+const defaultGraphOptions = {
+  scales: {
+    yAxes: [
+      {
+        ticks: {
+          beginAtZero: true,
```

Comment:
Is this necessarily the right default? Shouldn't it depend on the data? It could be a reasonable default for an empty graph, so maybe it belongs below.

Reply:
Without this option, the Y axis is pretty random (based on current values) and two bar graphs with very different values can look identical. I think it can be pretty confusing for students. Screenshot when this option is disabled:
<img width="949" alt="Screenshot 2020-11-20 at 12 48 15" src="https://user-images.githubusercontent.com/767857/99797027-c9db1400-2b2e-11eb-9d90-77c060541ad9.png">
Note that all the bars have the same height even though values are 2, 3, and 4. 
@lbondaryk, any opinion on that?

Reply:
I think I was imagining scatter plots when I made the comment. For bar charts I think you're right it is the appropriate default.